package.json

```diff
@@ -15,14 +15,16 @@
     "build-entypo": "./scripts/entypo.sh",
     "build-evilicons": "./scripts/evilicons.sh",
     "build-fontawesome": "./scripts/fontawesome.sh",
+    "build-fontawesome5": "./scripts/fontawesome5.sh",
     "build-feather": "./scripts/feather.sh",
     "build-foundation": "./scripts/foundation.sh",
     "build-ionicons": "./scripts/ionicons.sh",
     "build-materialicons": "./scripts/materialicons.sh",
     "build-materialcommunityicons": "./scripts/materialcommunityicons.sh",
     "build-octicons": "./scripts/octicons.sh",
     "build-zocial": "./scripts/zocial.sh",
-    "build-simplelineicons": "./scripts/simplelineicons.sh"
+    "build-simplelineicons": "./scripts/simplelineicons.sh",
+    "fa5upgrade": "./scripts/fa5upgrade.sh"
```

Comment:
This should probably exported as a `bin` instead of as a `script`.

Reply:
I'm not too familiar with this aspect of npm so I took a shot. Would adding it to ```bin``` make it runnable from the app folder (instead of navigating into the library folder for upgrading)? My original though was that you could just run something like ```npm run fa5upgrade``` from the top folder which would be easier (and cleaner).

Reply:
Yep, exactly :-)